### PR TITLE
RBE

### PR DESCRIPTION
**Pull request by zanderso**
See instructions on https://github.com/flutter/engine/pull/45271

### CODE REVIEW

File: build/rbe.cfg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+service=remotebuildexecution.googleapis.com:443
+instance=projects/flutter-rbe-prod/instances/default
+use_application_default_credentials=true
+enable_deps_cache=true
+xattr_digest=user.fuchsia.rbe.digest.sha256
+server_address=unix:///tmp/reproxy.sock
+log_dir=/tmp
```
[the remark this answers is not in the document]
Comment (zanderso) [reply build/rbe.cfg:7]: You tell me =) I just copied this from your doc.

File: build/toolchain/rbe.gni [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2013 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Defines the configuration of Goma.
```
[the remark this answers is not in the document]
Comment (zanderso) [reply build/toolchain/rbe.gni]: Done.

File: build/rbe.cfg [hunk 3]
```diff
@@ -0,0 +1,9 @@
+service=remotebuildexecution.googleapis.com:443
+instance=projects/flutter-rbe-prod/instances/default
+use_application_default_credentials=true
+enable_deps_cache=true
+xattr_digest=user.fuchsia.rbe.digest.sha256
```
[the remark this answers is not in the document]
Comment (zanderso) [reply build/rbe.cfg]: Done.

File: build/toolchain/android/BUILD.gn [hunk 4]
```diff
@@ -35,6 +36,9 @@ template("android_gcc_toolchain") {
     if (use_goma) {
       assert(!use_ccache, "Goma and ccache can't be used together.")
       compiler_prefix = "$goma_dir/gomacc "
+    } else if (use_rbe) {
+      assert(!use_goma)
+      compiler_prefix = "$rbe_dir/rewrapper --server_address=unix:///tmp/reproxy.sock --labels=type=compile,compiler=clang,lang=cpp --exec_root=$exec_root --platform=container-image=docker://gcr.io/cloud-marketplace/google/debian11@sha256:69e2789c9f3d28c6a0f13b25062c240ee7772be1f5e6d41bb4680b63eae6b304 "
```
[the remark this answers is not in the document]
Comment (zanderso) [reply build/toolchain/android/BUILD.gn]: These are now gn args declared in `rbe.gni`. I still need to update the linked `flutter/engine` PR to plumb them in the `gn` script.

File: build/rbe.cfg (hunk 1 above)
[the remark this answers is not in the document]
Comment (zanderso) [reply build/rbe.cfg:7]: Can you tell me how you plan to do that?

[the remark this answers is not in the document]
Comment (keyonghan) [reply build/rbe.cfg:7]: One way we can do is have recipes read this cfg file and override the `log_file` value before starting the bootstrap.

**Pull request merged**